src/lib/state/selectors/query.test.js

```diff
@@ -142,6 +142,23 @@ describe("queries with second level filters.", () => {
       ["type", "Publication"],
     ]);
   });
+
+  test("query with `subtype: Validation` should remove it from the correct type", () => {
```

Comment:
I think you test that you add `Validation` to the correct type here no? Either way, also testing the removal is good I think.

Reply:
Good point. Added.